stopcovid/dialog/models/state.py

```diff
@@ -48,6 +49,22 @@ class UserProfile:
     def __str__(self):
         return f"lang={self.language}, validated={self.validated}, " f"name={self.name}"
 
+    def json_serialize(self):
```

Comment:
Any reason not to use `UserProfileSchema.dumps` here?

Reply:
I'd actually just have a `to_dict` method that uses `UserProfileSchema.dump`. SqlAlchemy will handle dict->jsonb conversion.